jax/_src/numpy/polynomial.py

```diff
@@ -423,9 +423,65 @@ def polyval(p: Array, x: Array, *, unroll: int = 16) -> Array:
   y, _ = lax.scan(lambda y, p: (y * x + p, None), y, p, unroll=unroll)
   return y
 
-@implements(np.polyadd)
+
 @jit
 def polyadd(a1: Array, a2: Array) -> Array:
+  r"""Returns the sum of the two polynomials.
+
+  JAX implementation of :func:`numpy.polyadd`.
+
+  Args:
+    a1: Array of polynomial coefficients.
+    a2: Array of polynomial coefficients.
+
+  Returns:
+    An array containing the coefficients of the sum of input polynomials.
+
+  Note:
+
+    :func:`jax.numpy.polyadd` only accepts arrays as input unlike
+    :func:`numpy.polyadd` which accepts scalar inputs as well.
+
+  See also:
+    - :func:`jax.numpy.polysub`: Computes the difference of two polynomials.
+    - :func:`jax.numpy.polymul`: Computes the product of two polynomials.
+    - :func:`jax.numpy.polydiv`: Computes the quotient and remainder of polynomial
+      division.
+
+  Example:
+
+    ``jnp.polyadd`` does not accept scalar inputs:
+
+    >>> jnp.polyadd(2, 3)  # doctest: +IGNORE_EXCEPTION_DETAIL
```

Comment:
No reason to show this exception in my opinion. We've already noted this, and seeing the IndexError does not clarify anything for the reader.

Reply:
Removed.